src/ngraph/ngraph_emitter.cc

```diff
@@ -1261,6 +1262,21 @@ void Emitter::CreateLayerOps() {
 
     return mul_op;
   };
+  ngraph_op_funcs_["SequenceReverse"] =
+      [this](const NodePtr& node) -> NgraphNodePtr {
+    auto data = op_map_[node->inputs_[0]];
+    bool use_sequence_length = get_default(node, "use_sequence_length", false);
+    int seq_axis = get_default(node, "axis", 0);
+    const int batch_axis = 1;
```

Comment:
Coding style question: what's the purpose of making `batch_axis` const, but leaving the other local variables non-const?

Reply:
good point, I should make all parameters const.